utils/cliutils/commandsflags.go

```diff
@@ -2000,6 +2001,10 @@ var commandFlags = map[string][]string{
 		lcUrl, user, password, accessToken, serverId, lcDryRun, DistRules, site, city, countryCodes,
 		InsecureTls, CreateRepo, lcPathMappingPattern, lcPathMappingTarget,
 	},
+	ReleaseBundleExport: {
+		lcUrl, user, password, accessToken, serverId, lcDryRun, lcPathMappingTarget, lcPathMappingPattern, Project,
```

Comment:
Remove `lcDryRun` if not used (I don't think it is supported). Same for the rest unused flags.